<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UM Spool Maker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Material Library</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>23</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Material Filter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="material_filter">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>23</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>23</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Material Diameter</string>
     </property>
    </widget>
    <widget class="QComboBox" name="material_diameter">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>23</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>48</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Material</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_matSelector">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>48</y>
       <width>491</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>65</y>
       <width>621</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_brand">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>95</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_material">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>95</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>75</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Material Information</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_color">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>95</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_guid">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>125</y>
       <width>561</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>175</y>
      <width>621</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>NFC Tag</string>
    </property>
    <widget class="QLineEdit" name="line_nfcStatus">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>55</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Press 'Read Tag'</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>35</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Status</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_serial">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>55</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>35</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial Number</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>75</y>
       <width>621</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_cbrand">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>105</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Material</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_cmaterial">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>105</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_ccolor">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>105</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_cguid">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>135</y>
       <width>561</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_totalweight">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>185</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>165</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Weight (Current) [mg]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>185</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_remainweight">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>205</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_nweight">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>200</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>165</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Printing Time (h)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>205</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Remaining</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>180</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Weight (New) [mg]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_printtime">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>185</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_write">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Write Tag</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_read">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Tag</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>120</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_status">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Connecting to card reader...</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset_remain">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set remaining to new</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCommands_2">
    <property name="title">
     <string>Commands</string>
    </property>
    <addaction name="actionRescan_Materials"/>
    <addaction name="separator"/>
    <addaction name="actionRead_Tag"/>
    <addaction name="actionWrite_Tag"/>
   </widget>
   <addaction name="menuCommands"/>
   <addaction name="menuCommands_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRescan_Materials">
   <property name="text">
    <string>Rescan Materials</string>
   </property>
  </action>
  <action name="actionRead_Tag">
   <property name="text">
    <string>Read Tag</string>
   </property>
  </action>
  <action name="actionWrite_Tag">
   <property name="text">
    <string>Write Tag</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
